templates/webapi/main/index.html.ep

```diff
@@ -83,20 +83,11 @@
 
             <div class="form-group">
                 <strong>
-                    Auto refresh
-                    <%= help_popover('Help for <i>Auto refresh</i>' => '<p>Allows to update the data of this page automatically</p>') %>
+                    Auto-refresh interval
+                    <%= help_popover('Help for <i>Auto refresh</i>' => '<p>Allows to update the data of this page automatically. 0 means no auto-refresh</p>') %>
```

Comment:
With the placeholder text we don't see "0", right? So the additional sentence seems unnecessary.

Reply:
Yes, you're right, now it detects a 0 removing the content. So sure, this has to be removed

Reply:
done